WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -201,6 +201,29 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         createButtonCoordinator?.presentingTraitCollectionWillChange(traitCollection, newTraitCollection: newCollection)
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        guard let previousTraitCollection = previousTraitCollection,
+            let blog = blog else {
+            return
+        }
+
+        // When switching from compact width to regular width, if the dashboard tab is selected
+        // we must switch to the site menu tab.
+        //
+        // This ensures that the site menu is shown in the left pane of the split vc.
+        //
+        if previousTraitCollection.horizontalSizeClass == .compact,
+           traitCollection.horizontalSizeClass == .regular,
+           isShowingDashboard {
+            segmentedControl.selectedSegmentIndex = Section.siteMenu.rawValue
+            segmentedControlValueChanged()
+        }
```

Comment:
```suggestion
        } else if previousTraitCollection.horizontalSizeClass == .regular,
                  traitCollection.horizontalSizeClass == .compact {
            segmentedControl.selectedSegmentIndex = mySiteSettings.defaultSection.rawValue
            segmentedControlValueChanged()
        }
```

Wdyt about doing that? This ensures that when coming back to regular, the default tab will be shown (instead of the site menu every time).

Reply:
Fixed! 2c28308